### PR TITLE
Migrate more issues to SwiftUI

### PR DESCRIPTION
**Pull request by cselin**
TODO:
- [x] Debug Invalid returned hit test result for view in hierarchy when toggling on iPad which leads to untappable rows

|Active (Plain)|All (Plain)|All (Grouped)|All (InsetGrouped)|
|-|-|-|-|
|Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-11-03 at 10 45 29|Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-11-03 at 10 48 44|Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-11-03 at 10 48 09|Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-11-03 at 10 45 47|

### CODE REVIEW

**Comment by cselin** [comment]
@nickoneill let me know what you think about adding a toggle for showing all/active issues. Potentially could reproduce existing functionality of pushing another table for showing all with back button, but felt this was cleaner/easier.

**Comment by nickoneill** [comment]
I like the idea of unifying to a single screen for top and all issue, but a switch that's always visible is a little distracting imo. I prefer to keep folks focused on the main issues as they're the most time-sensitive ones available. Too much choice and it's harder to pick one to call on.

How about adding text "Show more/less issues" at the bottom of the list that scrolls with the content? For most people on phones you'll have to scroll to the bottom of the list before you see it.

**Comment by cselin** [comment]
@nickoneill what do you think about the styles? I'm leaning towards keeping plain for active issues and using grouped for all issues.

**Comment by cselin** [comment]
> TODO:
> 
> * [ ]  Debug Invalid returned hit test result for view in hierarchy when toggling on iPad which leads to untappable rows
> 

**TL;DR: @nickoneill in case you have an iPad running iPadOS 16+ see if toggling after selection borks the list.**


I don't have a new enough iPad to verify whether this is a simulator issue only. From what I can tell, toggling between active/all without a selection is fine, but once a selection current and then active/all is toggled I'm getting the following issue and the list is no longer usable from then out: 

```
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: 0x101e2e1b0>; hit-tested subview: <_TtGC7SwiftUI16PlatformViewHostGVS_P10$1c554ec9c17ListRepresentableGVS_28CollectionViewListDataSourceV9FiveCalls5Issue_GOS_19SelectionManagerBoxS4____: 0x101ebc9c0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIViewControllerWrapperView: 0x1040458f0>; hit-tested subview: <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: 0x101e2e1b0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UINavigationTransitionView: 0x101e27b30>; hit-tested subview: <UIViewControllerWrapperView: 0x1040458f0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UILayoutContainerView: 0x101e27780>; hit-tested subview: <UINavigationTransitionView: 0x101e27b30>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_UIPanelControllerContentView: 0x101e273e0>; hit-tested subview: <UILayoutContainerView: 0x101e27780>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_UISplitViewControllerPanelImplView: 0x101e26da0>; hit-tested subview: <_UIPanelControllerContentView: 0x101e273e0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_TtGC7SwiftUI16PlatformViewHostGVS_42PlatformViewControllerRepresentableAdaptorVS_28NavigationSplitRepresentable__: 0x101e26820>; hit-tested subview: <_UISplitViewControllerPanelImplView: 0x101e26da0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_TtGC7SwiftUI14_UIHostingViewGVS_15ModifiedContentV9FiveCalls14IssueSplitViewGVS_30_EnvironmentKeyWritingModifierGSqCS2_5Store____: 0x10401a4b0>; hit-tested subview: <_TtGC7SwiftUI16PlatformViewHostGVS_42PlatformViewControllerRepresentableAdaptorVS_28NavigationSplitRepresentable__: 0x101e26820>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIView: 0x1042053a0>; hit-tested subview: <_TtGC7SwiftUI14_UIHostingViewGVS_15ModifiedContentV9FiveCalls14IssueSplitViewGVS_30_EnvironmentKeyWritingModifierGSqCS2_5Store____: 0x10401a4b0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIDropShadowView: 0x104205160>; hit-tested subview: <UIView: 0x1042053a0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UITransitionView: 0x101e23180>; hit-tested subview: <UIDropShadowView: 0x104205160>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIWindow: 0x104307030>; hit-tested subview: <UITransitionView: 0x101e23180>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: 0x101e2e1b0>; hit-tested subview: <_TtGC7SwiftUI16PlatformViewHostGVS_P10$1c554ec9c17ListRepresentableGVS_28CollectionViewListDataSourceV9FiveCalls5Issue_GOS_19SelectionManagerBoxS4____: 0x101ebc9c0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIViewControllerWrapperView: 0x1040458f0>; hit-tested subview: <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: 0x101e2e1b0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UINavigationTransitionView: 0x101e27b30>; hit-tested subview: <UIViewControllerWrapperView: 0x1040458f0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UILayoutContainerView: 0x101e27780>; hit-tested subview: <UINavigationTransitionView: 0x101e27b30>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_UIPanelControllerContentView: 0x101e273e0>; hit-tested subview: <UILayoutContainerView: 0x101e27780>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_UISplitViewControllerPanelImplView: 0x101e26da0>; hit-tested subview: <_UIPanelControllerContentView: 0x101e273e0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_TtGC7SwiftUI16PlatformViewHostGVS_42PlatformViewControllerRepresentableAdaptorVS_28NavigationSplitRepresentable__: 0x101e26820>; hit-tested subview: <_UISplitViewControllerPanelImplView: 0x101e26da0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <_TtGC7SwiftUI14_UIHostingViewGVS_15ModifiedContentV9FiveCalls14IssueSplitViewGVS_30_EnvironmentKeyWritingModifierGSqCS2_5Store____: 0x10401a4b0>; hit-tested subview: <_TtGC7SwiftUI16PlatformViewHostGVS_42PlatformViewControllerRepresentableAdaptorVS_28NavigationSplitRepresentable__: 0x101e26820>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIView: 0x1042053a0>; hit-tested subview: <_TtGC7SwiftUI14_UIHostingViewGVS_15ModifiedContentV9FiveCalls14IssueSplitViewGVS_30_EnvironmentKeyWritingModifierGSqCS2_5Store____: 0x10401a4b0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIDropShadowView: 0x104205160>; hit-tested subview: <UIView: 0x1042053a0>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UITransitionView: 0x101e23180>; hit-tested subview: <UIDropShadowView: 0x104205160>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
Invalid returned hit test result for view in hierarchy: The returned hit test result is invalid and may cause issues as it is not in a window hierarchy: self: <UIWindow: 0x104307030>; hit-tested subview: <UITransitionView: 0x101e23180>; invalid returned view: <_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView: 0x104142da0>
```

I've tried resetting `selectedIssue = nil` upon toggle, but it didn't help. I'll continue trying to get to the bottom of this...

**Comment by nickoneill** [comment]
Yep, plain is fine. We can figure out how to add categories in the future if desired.
